Modules/Core/Common/include/itkBoundingBox.hxx

```diff
@@ -124,10 +124,10 @@ BoundingBox< TPointIdentifier, VPointDimension, TCoordRep, TPointsContainer >
   m_CornersContainer->clear();
   m_CornersContainer->Reserve(NumberOfCorners);
 
+  SizeValueType index = 0;
   for (const PointType& pnt: this->ComputeCorners())
     {
-    // Push back is not defined so we insert at the end of the list
-    m_CornersContainer->InsertElement(m_CornersContainer->Size(), pnt);
+    m_CornersContainer->SetElement(index++, pnt);
```

Comment:
For what it's worth, the original code from 2005 simply did `push_back(pnt)`!

https://github.com/InsightSoftwareConsortium/ITK/blob/423562511fff3e18284572c8fb267c16f249485e/Code/Common/itkBoundingBox.txx#L101 

But `SetElement` should also be OK in this case.